packages/lit-dev-content/site/blog/2022-01-05-lit-2.1-release.md

```diff
@@ -116,18 +116,20 @@ alternative to the positional argument API.
 private _icon!: Array<HTMLElement>;
 ```
 
-## Easy to use observers with @lit-labs/observer
+## Easy to use observers with @lit-labs/observers
 
-In Lit Labs we've introduced a new package called `@lit-labs/observer` which
+In Lit Labs we've introduced a new package called `@lit-labs/observers` which
 contains reactive controllers that let you easily use the web platform observer
 APIs: MutationObserver, IntersectionObserver, ResizeObserver, and
 PerformanceObserver. The controllers automatically drive the host element's
 update lifecycle when they observe changes.
 
 ```ts
+import { ResizeController } from '@lit-labs/observers/resize_controller.js';
```

Comment:
Curious how we landed on snake case for these files instead of kebab case. I went to double check in the source to make sure this wasn't a mistake.

Reply:
@sorvell might know that answer.